.circleci/config.yml

```diff
@@ -57,7 +56,6 @@ jobs:
           name: run tests
           command: |
             . env/bin/activate
-            pycodestyle --ignore=E402,W504 webcompat/ tests/ config/secrets.py.example
```

Comment:
The reason I removed this. It is because it is handled by `npm run lint`:

```
→ npm run lint

> webcompat@ lint /Users/karl/code/webcompat.com
> npm run lint:js && npm run lint:css && npm run lint:python


> webcompat@ lint:js /Users/karl/code/webcompat.com
> npx eslint ./Gruntfile.js ./tests ./grunt-tasks ./webcompat/static/js/lib


> webcompat@ lint:css /Users/karl/code/webcompat.com
> npx stylelint './webcompat/static/css/src/**/*.css' './webcompat/static/css/webcompat.dev.css'


> webcompat@ lint:python /Users/karl/code/webcompat.com
> pycodestyle --ignore=E402,W504 webcompat/ tests/ config/
```

Reply:
sgtm